.github/workflows/performance.yml

```diff
@@ -78,7 +78,6 @@ jobs:
         -DUMF_BUILD_LEVEL_ZERO_PROVIDER=ON
         -DUMF_BUILD_CUDA_PROVIDER=ON
         -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
-        -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=ON
```

Comment:
What if the build machine has not Internet access? If it is not possible to download jemalloc sources what happens?

Reply:
A user can provide its own local repo using `UMF_JEMALLOC_REPO`

Reply:
That's a good point. I think we should warn in the fetching steps and continue the build in such a case with `UMF_POOL_JEMALLOC_ENABLED=0`.